recipes/uvw/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "uvw" %}
+{% set version = "v2.12.1_libuv_v1.44" %}
+{% set sha256 = "3460842778e91e7d2fae4201e49e4521e9d94fbbf3891ae6c52d3c9fc0673598" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/skypjack/uvw/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake
+    - make  # [unix]
+    - {{ compiler('cxx') }}
+    - libuv
```

Comment:
Why is libuv required as a build dependency? That's very uncommon.

Reply:
You're right. Removed.